src/image/Image.cc

```diff
@@ -697,6 +698,20 @@ geom::Box2I bboxFromMetadata(daf::base::PropertySet & metadata)
     return geom::Box2I(xy0, dims);
 }
 
+template <typename T1, typename T2>
+bool imagesOverlap(ImageBase<T1> const& image1, ImageBase<T2> const& image2) {
+
+    auto arr1 = image1.getArray();
+    auto beg1Addr = reinterpret_cast<void const *>(arr1.front().begin());
+    auto end1Addr = reinterpret_cast<void const *>(arr1.back().end());
+
+    auto arr2 = image2.getArray();
+    auto beg2Addr = reinterpret_cast<void const *>(arr2.front().begin());
+    auto end2Addr = reinterpret_cast<void const *>(arr2.back().end());
+
+    return beg1Addr < end2Addr && beg2Addr < end1Addr;
```

Comment:
That sounds like a fatal flaw in any such test. Is it even safe to compare pointer identity? If so, I could at least check that one is not warping an exposure into self. If not, it may be time to give up on the test and just say "don't do that".

Reply:
I wonder if @TallJimbo will have an opinion on this.